### PR TITLE
[TM-359] Update README

### PR DESCRIPTION
**Pull request by rvem**
## Description
Problem: "Obtain binary or packages from CI" section in the README.md
isn't quite clear. Also, there are some inaccuracies in this file.

Solution: Update README, also update `packageRpm.nix` because it
contained some incorrect stuff.

## Related issue(s)

https://issues.serokell.io/issue/TM-359

#### Related changes (conditional)

- [x] I checked whether I should update the [README](../tree/master/README.md)

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).

### CODE REVIEW

**Comment by gromakovsky** [comment]
Well, https://github.com/serokell/tezos-client/issues/ is obviously not an issue :)

File: README.md [hunk 1]
```diff
@@ -19,13 +19,48 @@ Run `nix-build -A tezos-client-static -o tezos-static`
...
+sudo apt install <path to deb file>
+```
+
+### Use PPA with `tezos-client`
+
+Also if you are using Ubuntu you can use PPA in order to install `tezos-client`.
+In order to do that run the following commands:
+```
+sudo add-apt-repository ppa:serokell/tezos && sudo apt-get update
+sudo apt-get install tezos-client
```
Comment (gromakovsky) [inline README.md]: Have you tested these commands btw (I mean all commands present in this file)?

Comment (rvem) [reply README.md]: Everything except Fedora (I nuked VM with it in order to free some space for nix :(), but it this package is similar to one, I've been building earlier, so it should work as well

Comment (rvem) [reply README.md]: UPD: tested on Fedora as well

**Pull request merged**